code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1226,6 +1226,17 @@
 		RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_JANITOR
 	)
 
+/datum/design/borg_upgrade_uwu
+	name = "Cyborg UwU-speak 'Upgrade'"
```

Comment:
I think the single quotes might need an escape character here?
AKA: \'Upgrade\'"